src/app/search/search-filter-hashes.ts

```diff
@@ -199,16 +199,30 @@ export const emptySocketHashes = [
   791435474 // InventoryItem "Empty Activity Mod Socket"
 ];
 
-/** translate search terms to corresponding hashes */
-export const armorStatHashByName = {
+/** these stats actually exist on D2 armor */
+const realArmorStatHashByName = {
   mobility: 2996146975,
   resilience: 392767087,
   recovery: 1943323491,
   discipline: 1735777505,
   intellect: 144602215,
-  strength: 4244567218,
+  strength: 4244567218
+};
+/** these stats actually exist on DIM armor */
```

Comment:
???

Reply:
i'll clarify a bit more in comments. there's different groupings of stats based on what we want to do with them

"any" doesn't exist at all and is just included in filter name creation loops, total is worth including in the list for collecting max stats and does belong in a `name->hash` translator table. will rearrange a bit.

Reply:
I see, I got confused by the distinction between "these exist on D2 armor" vs. "these exist on DIM armor".